listings/models.py

```diff
@@ -1,3 +1,15 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Listing(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Shouldn't ids be part of listing and user model? From a db query perspective, having access to id should be helpful